src/modals/Synchronization/RunSynchronization.vue

```diff
@@ -1,6 +1,5 @@
 <script setup>
```

Comment:
Now we can test a synchronization via the run button, but we still have a test button in the action form

Reply:
And can we discuss all the other frontend changes?